languages/python/oso/oso/exceptions.py

```diff
@@ -4,9 +4,9 @@ class AuthorizationError(Exception):
 
```

Comment:
Three thoughts:

1. Seems kinda weird to have `OsoError` defined in `polar/exceptions.py` instead of `oso/exceptions.py`. Maybe it has to be that way for module dependency resolution purposes?
2. Should `AuthorizationError` inherit from `OsoError` instead of `Exception`?
3. Should the three errors currently in this file be called `OsoAuthorizationError`, `OsoNotFoundError`, and `OsoForbiddenError`, or is that obvious/redundant info?

Reply:
1. Yeah, I think it makes sense to put `OsoError` into `oso/exceptions.py`. We do have to import it into `polar/exceptions.py` but I think that's okay
2. Yeah
3. Ehh I don't think so -- none of our other errors are prefixed with `Oso-` and IMO it is a bit too verbose. Not many libraries I've seen prefix their classes with the library's name. Sometimes I've seen it recommended to import the namespace directly like this:
```
import oso
try:
  g.oso.authorize(...)
except oso.ForbiddenError:
  # forbidden :|
```

Reply:
Damn yeah unless we introduce a third `exceptions` module inside of the `languages/python/oso` folder, we can't put `OsoError` into `oso/exceptions.py` :(